diff --git a/ublox_dgnss_node/src/ubx_config_loader.cpp b/ublox_dgnss_node/src/ubx_config_loader.cpp
index b878655..f63fd8c 100644
--- a/ublox_dgnss_node/src/ubx_config_loader.cpp
+++ b/ublox_dgnss_node/src/ubx_config_loader.cpp
@@ -22,11 +22,7 @@
 #include <vector>
 #include <fstream>
 
-#ifdef ROS_DISTRO_humble
 #include <ament_index_cpp/get_package_share_directory.hpp>
-#else
-#include <ament_index_cpp/get_package_share_path.hpp>
-#endif
 
 namespace
 {
@@ -237,11 +233,7 @@ ubx_cfg_item_map_t UbxConfigLoader::load_from_toml(
 
 std::string UbxConfigLoader::get_default_toml_path(const std::string & device_family)
 {
-  #ifdef ROS_DISTRO_humble
   auto package_share = ament_index_cpp::get_package_share_directory("ublox_dgnss");
-  #else
-  auto package_share = ament_index_cpp::get_package_share_path("ublox_dgnss").string();
-  #endif
 
   std::string family_lower = device_family;
   std::transform(
